modules/@apostrophecms/ui/ui/apos/components/AposTagList.vue

```diff
@@ -40,10 +41,20 @@ export default {
       }
     };
   },
+  watch: {
+    active (newValue) {
+      this.$emit('update', [ ...newValue ]);
```

Comment:
`[ ...javascriptIsConfusing ]`

Reply:
why not `this.$emit('update', newValue);`

Reply:
I originally did that, but I was getting a bug where the filter value in AposMediaManager was updated _before_ the method was called to update it. Something to do with it being the literal same array that was updated here in the tag list. By making it into a new array each time we're safely passing the new values.

Reply:
Makes sense